content/docs/sidebar.json

```
@@ -544,6 +544,17 @@
               "share-a-project"
             ]
           },
+          {
+            "slug": "model-registry",
+            "source": false,
+            "children": [
+              "what-is-a-model-registry",
```

Nav fix (committing)
```suggestion
              {
                "label": "What is a Model Registry",
                "slug": "what-is-a-model-registry"
              }
```